torch/nn/functional.py

```diff
@@ -2815,6 +2815,26 @@ def multi_margin_loss(input, target, p=1, margin=1., weight=None, size_average=N
     torch.Size([1, 1, 12, 12])
 """)
 
+pixel_unshuffle = _add_docstr(torch.pixel_unshuffle, r"""
+pixel_unshuffle(input, downscale_factor) -> Tensor
+
+Rearranges elements in a tensor of shape :math:`(*, C, H \times r, W \times r)` to a
+tensor of shape :math:`(*, C \times r^2, H, W)`.
+
+See :class:`~torch.nn.PixelUnshuffle` for details.
```

Comment:
It always seems like an interesting quirk to me that we refer to the module over the function as our source of truth, but I think this is how most of our nn documents work.